models/handlers.go

```diff
@@ -5,7 +5,7 @@ import (
 
 	"time"
 
-	"github.com/vmihailenco/taskq"
+	"github.com/vmihailenco/taskq/v3"
```

Comment:
Fingers crossed.

Reply:
Not working! But this could be due to the deprecations as we have been using a much older version of taskq and we have jumped with over 3 major version with this commit.

Reply:
@kushthedude , try `go get -u github.com/vmihailenco/taskq/v3@master` since this issue was fixed recently only....

Reply:
> this could be due to the deprecations as we have been using a much older version of taskq and we have jumped with over 3 major version with this commit.

That is the reason why static lint is failing.

Reply:
> @kushthedude , try `go get -u github.com/vmihailenco/taskq/v3@master` since this issue was fixed recently only....

0.8 was released 21 days ago & this was fixed on 27 march.